src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -1175,6 +1177,27 @@ class MacroAssembler: public Assembler {
   //     be used instead.
   //     All instructions are embedded at a call site.
   //
+  //   - indirect call: movptr + jalr
+  //     This too can reach anywhere in the address space, but it cannot be
+  //     patched while code is running, so it must only be modified at a safepoint.
+  //     This form of call is most suitable for targets at fixed addresses, which
+  //     will never be patched.
+  //
+  //   - patchable far call:
+  //     This is only available in C1/C2-generated code (nmethod).
+  //
+  //     [Main code section]
+  //       auipc
+  //       ld <address_from_stub_section>
+  //       jalr
+  //     [Stub section]
+  //     trampoline:
+  //       <64-bit destination address>
+  //
+  //    To change the destination we simply atomically store the new
+  //    address in the stub section.
+  //
+  // Old patchable far calls: (-XX:+UseTrampolines)
   //   - trampoline call:
```

Comment:
How about combine the two lines? Like:
`- trampoline call (old patchable far call / -XX:+UseTrampolines):`

Reply:
ok